src/index.js

```diff
@@ -68,6 +67,10 @@ export default class List extends React.Component {
 
   getWrapper = () => ReactDOM.findDOMNode(this.listWrapper);
 
+  getDefaultHeightWidth = () => {
+    return this.props.className ? {} : { height: '300px', width: '600px' };
```

Comment:
Let's remove the redundant `return` and braces here.
```js
getDefaultHeightWidth = () => this.props.className ? {} : { height: '300px', width: '600px' };
```

Also why are we setting arbitrary values in the `height` and `width` props? Let's set them to `100%`. This way, the component height and width will default to using the full dimensions of its parent.

The user has to be able to use the component with as little configuration as possible. So setting a container width and height in the classname that they have to also set would be quite a lot of initial work.